mocks/mocks.go

```diff
@@ -41,10 +48,25 @@ func (m *Mocks) Start() error {
 	return nil
 }
 
+// Maybe slow after go1.15 for gracefull shutdown reason
+// https://github.com/golang/go/blob/release-branch.go1.15/src/net/http/server.go#L2679
+// See ShutdownContext for workaround
 func (m *Mocks) Shutdown() {
+	_ = m.ShutdownContext(context.TODO())
```

Comment:
I guess here we should pre-cancel the context to defeat the slowing down issue.

Reply:
I planned to switch my tests to context-related method, and do it in my code, but ok, lets leave "old" behaviour here.